lib/bitmap/bitmap.nit

```diff
@@ -141,7 +138,6 @@ class Bitmap
 		# =============== DIB header ================
 		for x in [0..39] do
 			var b = fileReader.read_byte
-			if b == null then return
 			dib_header[x] = b.to_i
```

Comment:
Same here

Reply:
I only removed the warning, I will let you make the fixes.